### PR TITLE
Update "update_package_versions" script

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Update "update_package_versions" script

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments
n/a

### CODE REVIEW

**Comment by marcofavorito** [comment]
- [ ] files are saved without new lines.
- [ ] the interactive mode should only be used if there is uncertainty about the package type. First, load all package identifiers and see where there are multiple package types with same public id, only those need interactive mode. Then, even for those, if the add or fetch command is used we know the type of package, so no need for user confirmation.
- [ ] references in package/README.md files don’t get updated.

File: docs/ml-skills.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ cd ml_data_provider
 aea add connection fetchai/p2p_libp2p:0.8.0
 aea add connection fetchai/soef:0.8.0
 aea add connection fetchai/ledger:0.4.0
-aea add skill fetchai/ml_data_provider:0.10.0
```
Comment (DavidMinarsch) [inline docs/ml-skills.md:80]: This appears to be an error!

File: packages/fetchai/agents/ml_data_provider/aea-config.yaml [hunk 2]
```diff
@@ -19,8 +19,8 @@ protocols:
 - fetchai/oef_search:0.6.0
 skills:
 - fetchai/error:0.5.0
-- fetchai/generic_seller:0.11.0
-- fetchai/ml_data_provider:0.10.0
+- fetchai/generic_seller:0.12.0
+- fetchai/ml_data_provider:0.12.0
```
Comment (DavidMinarsch) [inline packages/fetchai/agents/ml_data_provider/aea-config.yaml:23]: error

File: tests/test_docs/test_bash_yaml/md_files/bash-ml-skills.md [hunk 3]
```diff
@@ -9,12 +9,12 @@ cd ml_data_provider
 aea add connection fetchai/p2p_libp2p:0.8.0
 aea add connection fetchai/soef:0.8.0
 aea add connection fetchai/ledger:0.4.0
-aea add skill fetchai/ml_data_provider:0.10.0
+aea add skill fetchai/ml_data_provider:0.12.0
```
Comment (DavidMinarsch) [inline tests/test_docs/test_bash_yaml/md_files/bash-ml-skills.md:12]: error

File: tests/test_docs/test_orm_integration/test_orm_integration.py [hunk 4]
```diff
@@ -334,4 +334,4 @@ def test_strategy_consistency():
                 "Code block not present in strategy file:\n{}".format(
                     python_code_block["text"]
                 )
-            )
+            )
```
Comment (DavidMinarsch) [inline tests/test_docs/test_orm_integration/test_orm_integration.py:337]: It should not remove the line here!

File: tests/test_packages/test_skills/test_ml_skills.py [hunk 5]
```diff
@@ -80,7 +80,7 @@ def test_ml_skills(self, pytestconfig):
         self.add_item("connection", "fetchai/soef:0.8.0")
         self.set_config("agent.default_connection", "fetchai/p2p_libp2p:0.8.0")
         self.add_item("connection", "fetchai/ledger:0.4.0")
-        self.add_item("skill", "fetchai/ml_data_provider:0.10.0")
+        self.add_item("skill", "fetchai/ml_data_provider:0.12.0")
```
Comment (DavidMinarsch) [inline tests/test_packages/test_skills/test_ml_skills.py:83]: error

File: tests/test_packages/test_skills/test_ml_skills.py [hunk 6]
```diff
@@ -257,13 +257,13 @@ def test_ml_skills(self, pytestconfig):
...
```
Comment (DavidMinarsch) [inline tests/test_packages/test_skills/test_ml_skills.py:260]: error

File: tests/test_packages/test_skills/test_thermometer.py [hunk 7]
```diff
@@ -402,4 +402,4 @@ def test_thermometer(self):
         assert (
             self.is_successfully_terminated()
         ), "Agents weren't successfully terminated."
-        wait_for_localhost_ports_to_close([9000, 9001])
+        wait_for_localhost_ports_to_close([9000, 9001])
```
Comment (DavidMinarsch) [inline tests/test_packages/test_skills/test_thermometer.py:405]: empty line removed

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**